### PR TITLE
fix: model onReady method fires at wrong moment

### PR DESCRIPTION
**Pull request by bigkrp**
resolving [this](https://github.com/reearth/resium/issues/621) issue

### CODE REVIEW

File: src/Model/Model.ts [hunk 1]
```diff
@@ -81,6 +81,8 @@ const cesiumReadonlyProps = [
 
 export const otherProps = ["onReady", "onError", "url"] as const;
 
+let readyEventListener: ((model: CesiumModel) => void) | null = null
```
Comment (keiya01) [inline src/Model/Model.ts:84]: This will be overridden whenever Model component is created.

**Review by keiya01 — COMMENTED** [review]
Thank you for your PR. Please check this comment:pray:

**Comment by bigkrp** [comment]
@keiya01 used state for the listener, please review changes

**Comment by bigkrp** [comment]
Please review

**Pull request merged**